apps/member/utils.py

```diff
@@ -38,6 +40,47 @@ def get_member_data(requesting_user, member, resource_name, record_type):
             raise Http404
         else:
             resource_grant = resource_grants.first()
-        data = resource_grant.resource_class(resource_grant.member).get(record_type)
+        response = resource_grant.resource_class(resource_grant.member).get(record_type)
```

Comment:
Should handle the case where the member hasn't hooked up their access to hixny yet.

This function could through something like a notfound exception or a custom exception which can then be observed and handled at the view level.

Reply:
@whytheplatypus Since @dchukhin is out for a while, could you make the requested changes and merge?  Id like to demonstrate on Friday and next week.